README.md

```diff
@@ -559,6 +559,9 @@ $client->customers()->magentoLegacyKeys()->remove($customerId, $publicKey);
 ```
 
 ### Validate incoming webhook payloads
+
+When you create or update a webhook in Private Packagist an optional secret can be set. This secret gets used to create a hash signature which is sent with each request in the headers as `Packagist-Signature`. The secret and signature can then be used on your server to validate that the reuqest was made by Private Packagist.
```

Comment:
Please specify that if no secret is set, the signature will not be set and cannot be verified.

Also is it a hash or a signature? those are kind of two separate things?

Typo in "request" (reuqest)